repose-aggregator/components/services/datastore-service/datastore-service-api/src/main/resources/META-INF/schema/config/hazelcast-datastore.xsd

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+  Repose
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Copyright (C) 2010 - 2015 Rackspace US, Inc.
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+       http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+           xmlns="http://docs.openrepose.org/repose/hazelcast-datastore/v1.0"
+           targetNamespace="http://docs.openrepose.org/repose/hazelcast-datastore/v1.0"
+           attributeFormDefault="unqualified"
+           elementFormDefault="qualified">
+
+    <xs:element name="hazelcast-datastore" type="HazelcastDatastoreConfig"/>
+
+    <xs:complexType name="HazelcastDatastoreConfig">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Hazelcast Datastore settings.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:choice>
+            <xs:element name="standard" type="StandardConfig"/>
```

Comment:
I don't really like this name. Maybe full or custom?

Reply:
I chose "standard" since this choice enables the usage of one of the official, standard methods of configuring Hazelcast. I grant you, however, that a standard Hazelcast configuration is not necessarily a standard Hazelcast Datastore configuration.

As for alternatives, I dislike "custom" very much in this case. Any configuration makes for a custom deployment. I suppose I'm fine with "full". Or maybe "traditional"?